conf/config-default.yaml

```diff
@@ -205,8 +206,8 @@ plugins:                          # plugin list (sorted in alphabetical order)
   - consumer-restriction
   - cors
   - echo
-  #- error-log-logger
-  #- example-plugin
+  # - error-log-logger
+  # - example-plugin
```

Comment:
Need to update this line: https://github.com/apache/apisix/blob/f476d2c3dc68439406a9531fc8b4a2a2d50c1333/t/APISIX.pm#L75